### ISSUE TITLE
Managesieve plugin: Code syntax highlighting not working in Roundcube Dependent versions

### ISSUE THREAD

**Issue by Bojo78** [roundcube/roundcubemail#6930]
Roundcube: 1.3 Dependent, 1.4 Dependent
OS: CentOS 6.10, PHP 5.6.40/Apache

Code syntax highlighting in Edit filter set textarea in Settings->Filters doesn’t work in Roundcube Dependent versions, due to incorrect folder permissions. 

Apache error logs:
`(13)Permission denied: /var/www/roundcubemail/plugins/managesieve/codemirror/.htaccess pcfg_openfile: unable to check htaccess file, ensure it is readable`

Plugin folder permissions:
```
drwxr-xr-- 5 root root  4096 Sep 18 13:03 codemirror
drwxr-xr-x 4 root root  4096 Sep 18 12:43 helpdocs
drwxr-xr-x 3 root root  4096 Sep 18 12:43 lib
drwxr-xr-x 2 root root  4096 Aug 28 14:24 localization
drwxr-xr-x 4 root root  4096 Sep 18 12:43 skins
```

Changing permissions of /var/www/roundcubemail/plugins/managesieve/codemirror/ folder and subfolders to 0755 solves the problem.

**Comment by alecpl** [comment]
I don't know how did you install Roundcube, I cannot reproduce. What I did:
1. Donwloaded https://github.com/roundcube/roundcubemail/releases/download/1.4-rc2/roundcubemail-1.4-rc2.tar.gz and extracted
2. Executed bin/install-jsdeps.sh

The result is:
```
drwxrwxr-x 5 alec alec  4096 Sep 23 09:07 codemirror
```
And there's no .htaccess file in that folder.

**Comment by Bojo78** [comment]
I’ve installed Roundcube the same way, but 1.3.10 version. 
There's no .htaccess file in codemirror folder, but Apache is looking for it when I try to edit a filter set and an error is generated. 
I installed Roundcube 1.4 RC2 and there is no problem with plugin folder permissions. 

Comparing the script install-jsdeps.sh in 1.4 RC2 and 1.3, I found the problem is lines 198 and 234 in 1.3 version. 

Changing them to  
`mkdir($destdir, 0775, true); `
and 
`mkdir($parentdir, 0775, true);`
corrects the problem.

Initially, I thought version 1.4 was also affected because I was upgrading from 1.3.
